preliz/maxent.py

```diff
@@ -65,9 +68,15 @@ def maxent(
     # Heuristic to provide an initial guess for the optimization step
     # We obtain those guesses by first approximating the mean and standard deviation
     # from intervals and mass and then use those values for moment matching
-    distribution._fit_moments(  # pylint:disable=protected-access
-        mean=(lower + upper) / 2, sigma=((upper - lower) / 4) / mass
-    )
+    mean = (lower + upper) / 2
+    sigma = ((upper - lower) / 4) / mass
+
+    if "exgaussian" in distribution.name:
+        distribution._fit_moments(  # pylint:disable=protected-access
+            mean, sigma, skewness=2 * nu**3 / (sigma**2 + nu**2) ** 1.5
+        )
```

Comment:
same as before, maxent should be agnostic of the details of particular distributions.